doc/hotspot-style.html

```diff
@@ -94,6 +94,7 @@ <h3 id="source-files">Source Files</h3>
 <li><p>All source files must have a globally unique basename. The build system depends on this uniqueness.</p></li>
 <li><p>Do not put non-trivial function implementations in .hpp files. If the implementation depends on other .hpp files, put it in a .cpp or a .inline.hpp file.</p></li>
 <li><p>.inline.hpp files should only be included in .cpp or .inline.hpp files.</p></li>
+<li><p>All .inline.hpp files should include their corresponding .hpp file as the first include line. Externally used declarations should be put in the .hpp file, and not in the the .inline.hpp file. This rule exists to resolve problems with circular dependencies between.inline.hpp files.</p></li>
```

Comment:
typo: the the
Need space after between on last line

Reply:
Fixed.